packages/vendure-plugin-accept-blue/src/api/api-extensions.ts

```diff
@@ -118,6 +118,10 @@ export const commonApiExtensions = gql`
     savedAcceptBluePaymentMethods: [AcceptBluePaymentMethod!]!
   }
 
+  extend enum HistoryEntryType {
+    ACCEPT_BLUE_TRANSACTION_UPDATE
```

Comment:
We need a custom Admin UI component for this, or otherwise it throws an error in Admin Ui right? Or am I mistaken?

If we do, it might be easier to just add it as order note, or mis-use any of the other built in entries.

Reply:
Ah yeah, The f**** angular crap. I don't work with that.